coalib/bearlib/abstractions/Linter.py

```diff
@@ -775,9 +775,8 @@ def linter(executable: str,
     ...         output_format='regex',
     ...         output_regex='...')
     ... class YLintBear:
-    ...     @staticmethod
-    ...     def create_arguments(config_file):
-    ...         return '--lint', filename
+    ...     def create_arguments(self, config_file):
```

Comment:
I don't think the decorator for static method should be removed as it should still be a static method. Anything I'm missing here @Makman2?

Reply:
> access to the file dict sadly happens via self.file_dict for a GlobalBear... this is gonna change in next gen :thumbsup: